static/js/containers/TermsPage.js

```diff
@@ -0,0 +1,605 @@
+                will provide that member with written notice of such claim, suit
+                or action, at the email address provided by that member at the
+                time of registration.
+              </div>
+              <h3 className="terms-subheading">Grant Of License</h3>
+              <div className="terms-div">
+                MIT does not claim ownership of the materials you post, upload,
+                input or submit to the ODL Video site. However, by posting,
+                uploading, inputting, providing or submitting your content to
+                ODL Video, you are granting MIT, its affiliated organizations
+                and partners, a worldwide, irrevocable, royalty-free,
+                non-exclusive, sub-licensable license to use, reproduce, create
+                derivative works of, distribute, publicly perform, publicly
+                display, transfer, transmit, distribute and publish that content
+                for the purposes of displaying that content on ODL Video or for
+                any other educational or non-commercial use of that content.
+              </div>
+              <div className="terms-div">
+                In addition, when you upload or post content to the ODL Video
+                site, you grant MIT a license to distribute that content, either
+                electronically or via other media, to users seeking to download
+                it through the ODL Video site or for purposes of other services
+                provided by ODL Video and to display such content on MIT
+                affiliated sites. This license shall apply to the distribution
+                and the storage of your content in any form, medium, or
+                technology now known or later developed. By posting, uploading,
+                inputting, providing or submitting your content, you warrant and
+                represent that you own or otherwise control all of the rights to
+                your content, including without limitation, all the rights
+                necessary for you to provide, post, upload, input or submit the
+                content and for ODL Video to post, upload, cross-post, or
+                cross-upload the content.
+              </div>
+              <div className="terms-div">
+                You may remove content you have posted on ODL Video at any time.
+                When you delete content from ODL Video, such deleted content,
+                while not available to the viewing public and other ODL Video
+                users, will remain on the ODL Video server until such time as
+                you make a specific request to ODL Video for permanent deletion
+                of such content from the ODL Video server. Such requests must be
+                made in writing, via email, to ODL Video customer service at the
+                following address: odl-video-support@mit.edu. When you do remove
```

Comment:
Could you link this email like you did earlier in this doc?

Reply:
✅